### PR TITLE
Adds mapping all audio to access script

### PR DESCRIPTION
**Pull request by ablwr**
Connected to https://github.com/amiaopensource/ffmprovisr/issues/383

@bturkus I can't request a review from you (???) but I'd like one.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -294,15 +294,18 @@ <h3>Transcode into a deinterlaced Apple ProRes LT</h3>
     <input type="checkbox" id="transcode_h264">
     <div class="hiding">
       <h3>Transcode to H.264</h3>
-      <p><code>ffmpeg -i <em>input_file</em> -c:v libx264 -pix_fmt yuv420p -c:a aac <em>output_file</em></code></p>
+      <p><code>ffmpeg -i <em>input_file</em> -c:v libx264 -pix_fmt yuv420p -c:a
+      aac -map 0:a <em>output_file</em></code></p>
```
Comment (retokromer) [inline index.html]: I would not add a hardcoded newline.

File: index.html [hunk 2]
```diff
@@ -294,15 +294,18 @@ <h3>Transcode into a deinterlaced Apple ProRes LT</h3>
...
       <dl>
         <dt>ffmpeg</dt><dd>starts the command</dd>
         <dt>-i <em>input_file</em></dt><dd>path, name and extension of the input file</dd>
         <dt>-c:v libx264</dt><dd>tells FFmpeg to encode the video stream as H.264</dd>
         <dt>-pix_fmt yuv420p</dt><dd>libx264 will use a chroma subsampling scheme that is the closest match to that of the input. This can result in Y′C<sub>B</sub>C<sub>R</sub> 4:2:0, 4:2:2, or 4:4:4 chroma subsampling. QuickTime and most other non-FFmpeg based players can’t decode H.264 files that are not 4:2:0. In order to allow the video to play in all players, you can specify 4:2:0 chroma subsampling.</dd>
         <dt>-c:a aac</dt><dd>encode audio as AAC.<br>
-          AAC is the codec most often used for audio streams within an .mp4 container.</dd>
+          AAC is the codec most often used for audio streams within an .mp4
+          container.</dd>
```
Comment (retokromer) [inline index.html]: ditto

**Review by retokromer — APPROVED** [review]
Two stylistic nits, otherwise LGTM. Thank you!

**Comment by ablwr** [comment]
My editor autocuts to ~80chars.

**Comment by retokromer** [comment]
(At 72 sharp, if it’s a good old Cobol editor ;-)

**Comment by ablwr** [comment]
I patched

**Pull request merged**